addons/account/wizard/account_payment_register.py

```diff
@@ -1273,7 +1273,7 @@ def _create_payments(self):
 
         wizard = self.sudo() if from_sibling_companies else self
 
-        payments = wizard._init_payments(to_process, edit_mode=edit_mode)
+        payments = wizard.with_context(default_partner_bank_id=False)._init_payments(to_process, edit_mode=edit_mode)
```

Comment:
Isn't the partner bank set in the `create_vals` in `to_process`?
Why does the context matter here?